modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/ColumnFamilyUtils.java

```diff
@@ -49,7 +54,7 @@ class ColumnFamilyUtils {
      * Utility enum to describe a type of the column family - meta or partition.
      */
     enum ColumnFamilyType {
-        META, PARTITION, HASH_INDEX, SORTED_INDEX, UNKNOWN;
+        META, PARTITION, GC_QUEUE, HASH_INDEX, SORTED_INDEX, UNKNOWN;
```

Comment:
Just thoughts: why not write down the names in the enumeration, for example, `GC_QUEUE("cf-gc")`, or use the enum name as the name for the cf.

Reply:
We can do this, but we can't use enum name as CF name, because the default one comes from the RocksDB itself

Reply:
Ok